backend/src/middleware/notifications/mentions-in-grops.spec.ts

```diff
@@ -0,0 +1,845 @@
+import { createTestClient } from 'apollo-server-testing'
```

Comment:
I like the spit of the tests but I would wish for a prefix of the filename where the tests resides in. This is currently in a subfolder with one resolver, which makes this possible, but thats flawed in itself.

Reply:
I do not understand. What do you want, can you give an example?

Reply:
id like the tests to be named `notificationsMiddleware.*.spec.ts` respective `notificationsMiddleware.mentions-in-grops.spec.ts` to ensure that the files can be associated with the actual file to be tested even when moved into another folder with more files.

Reply:
btw `grops` should be `groups`